### PR TITLE
Relay: remove default environment factory

### PR DESCRIPTION
**Pull request by mrtnzlml**
Cleanup. This way we have universally usable QR across companies.

### CODE REVIEW

File: src/relay/README.md [hunk 1]
```diff
@@ -88,9 +89,7 @@ function handleResponse(props: AppQueryResponse) {
 export default function App(props) {
   return (
     <QueryRenderer
-      // Following `clientID` helps us to identify who is sending the request and it's
-      // required unless you provide custom `environment` (see below).
-      clientID="unique-client-identification"
+      environment={ ... } // see bellow
```
Comment (tbergq) [inline src/relay/README.md]: Your most common mistake here 🙈 , `below`

Comment (mrtnzlml) [reply src/relay/README.md]: omg 😂

Comment (tbergq) [reply src/relay/README.md]: I fixed it for you ;)

Comment (tbergq) [reply src/relay/README.md]: Oh, seems you did too 🙈

Comment (mrtnzlml) [reply src/relay/README.md]: Thanks! :)

**Pull request merged**